NEWS

```diff
@@ -208,6 +208,10 @@ NEWS - user visible changes				-*- outline -*-
    be reported and recovered from.  This is configurable with option
    `missing-period`.
 
+   If not specified, the compiler tries automatically recognize the format,
+   using either fixed or free, depending on column 7 in the first non-space
+   line. This feature can be disabled using `-fformat=noauto`.
```

Comment:
Do we need noauto? If in question someone would explicit specify -fformat=fixed (for the old behavior), no? In this case we can drop the noauto completely.

Reply:
Indeed, it is equivalent to `fixed`, I will remove it.